servicepulse/intro-archived-messages.md

```diff
@@ -4,28 +4,32 @@ summary: Describes how ServicePulse monitors archived failed messages, and allow
 component: ServicePulse
 tags:
  - ServicePulse
-reviewed: 2016-04-06
+reviewed: 2016-12-15
 ---
 
 
-In Versions 1.5.0 and above screens to manage archived messages have been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal.
+In Versions 1.5.0 and above a screen to manage archived messages has been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal.
 
-ServiceControl version 1.13.0 and above has a configurable [ErrorRetentionPeriod](/servicecontrol/creating-config-file.md) that schedules the disposal of archived failed messages.
-
-Archived messages can be found by going to the failed messages screen.
+Archived Messages can be found in a tab in the Failed Messages page. From the Failed Messages page, messages can also be archived from the Failed Groups tab, from the list of message corresponding to a certain Failed Group, and from the All Messages view.
```

Comment:
That's a really complex paragraph ;) can we simplify? e.g. like "Archived Messages can be accessed/viewed in the Failed Messages page (... \ tab xyz), Failed Groups page (...path), and in the All Messages view (how?).